.github/workflows/stale.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Stale issue handler'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 7 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4.0.0
+        id: stale
+        with:
+          stale-issue-message: 'This Issue will be closed in 30 days. Please remove the "Stale" label or comment to avoid closure with no action.'
+          stale-pr-message: 'This PR will be closed in 30 days. Please remove the "Stale" label or comment to avoid closure with no action.'
+          days-before-stale: 300
+          days-before-close: 30
+          stale-issue-label: 'stale'
+          stale-pr-label: 'stale'
+          close-issue-label: 'closed'
+          close-pr-label: 'closed'
```

Comment:
I would add a message instead on automatic close. See [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-python/blob/main/.github/workflows/close-stale-issues.yml#L14)

Reply:
We already have a message for when it goes stale, which when closed would be the last message. So I don't think that it's necessary...

I think that by using the label it's a bit more concise and also more searchable without getting false positive matches.